src/systems/filecoin_vm/sysactors/init_actor.go

```diff
@@ -58,7 +58,7 @@ func (a *InitActorCode_I) Constructor(rt Runtime) InvocOutput {
 	h := rt.AcquireState()
 	st := &InitActorState_I{
 		AddressMap_: map[addr.Address]addr.ActorID{}, // TODO: HAMT
-		NextID_:     addr.ActorID(0),
+		NextID_:     addr.ActorID(addr.FirstNonSingletonActorId),
```

Comment:
Shouldn't this get incremented afterward?

Reply:
This is just initialization. `Exec` increments it when a new actor is created.